tests/drivers/dma/chan_link_transfer/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2020 STMicroelectronics
```

Comment:
wrong company and year?

Reply:
I copied the file from one of the other dma driver tests.